apps/climatemappedafrica/src/components/DataIndicators/useStyles.js

```diff
@@ -0,0 +1,151 @@
+import makeStyles from "@mui/styles/makeStyles";
+
+const useStyles = makeStyles(({ typography, breakpoints, palette }) => ({
```

Comment:
With exception of `section` (which comes from the old `commons-ui/core`, I think all these `useStyles` can easily be converted into [sx](url) on component(s).